releases/R2023b/README.md

```diff
@@ -43,8 +43,8 @@ To deploy MATLAB Production Server onto an existing VPC, specify these additiona
 | Parameter Name | Value |
 |--------------- | ----- |
 ||**Network License Manager**|
-| **Deploy License Server** | Specify whether you want to deploy the Network License Manager for MATLAB. This parameter is available only if you use the deployment template for an existing VPC. <p>If you are using an existing VPC, see the [Use Existing License Server in Existing VPC](#use-existing-license-server-in-existing-vpc) section.</p> |
-|**IP Address and port number of Existing Network License Manager**|<p> Optional parameter: Specify the port number and private DNS name or private IP address of the network license manager that has already been deployed to the existing VPC. Specify it in the format <port>@<privateDNSname>, for example, `27000@ip-172-30-1-89.ec2.internal` or `27000@172.30.1.89`. By default, the license manager uses port 27000. Leave this field blank if you are deploying a new network license manager.</p>|
+| **Deploy License Server** | Optional parameter: Specify whether you want to deploy the Network License Manager for MATLAB. This parameter is available only if you use the deployment template for an existing VPC. <p>Instead of specifying values for the following two parameters, you may choose to configure the License Server manually. For manual configurtion steps for using an existing License Server with an existing VPC, see the [Use Existing License Server in Existing VPC](#use-existing-license-server-in-existing-vpc) section.</p> |
```

Comment:
"Deploy License Server" is NOT an optional parameter - Users must choose whether they want to deploy a license server or not. The following two parameters are optional. 
After you address this change, you can Squash and Merge the staging branch into master. Let me know if you need any help doing that.
Also, if you have not already done so, you can try deploying MPS using an existing VPC and try out these new parameters.

Reply:
Squashed and merged.